virtool/samples/api.py

```diff
@@ -356,7 +348,7 @@ async def job_remove(req):
 
 @routes.view("/samples/{sample_id}/analyses")
 class AnalysesView(PydanticView):
-    async def get(self) -> Union[r200[List[GetSampleAnalysesResponse]], r403, r404]:
+    async def get(self, sample_id: str, /, term: Optional[str]) -> Union[r200[List[GetSampleAnalysesResponse]], r403, r404]:
```

Comment:
Use `Field` to add a description so this can be shown in the docs.

Write a description similar to that for labels. Check the pagination query for the actual fields we are text matching.